src/libmongoc/src/mongoc/mongoc-cluster-private.h

```diff
@@ -44,7 +44,6 @@ BSON_BEGIN_DECLS
 typedef struct _mongoc_cluster_node_t {
    mongoc_stream_t *stream;
    char *connection_address;
-   uint32_t generation;
```

Comment:
A `mongoc_cluster_node_t` stores an application connection on a pooled client. The `handshake_sd` already contained the `generation` associated with the connection. This was a redundant copy of that value.